libs/community/langchain_community/embeddings/yandex.py

```diff
@@ -48,12 +47,12 @@ class YandexGPTEmbeddings(BaseModel, Embeddings):
     api_key: SecretStr = ""  # type: ignore[assignment]
     """Yandex Cloud Api Key for service account
     with the `ai.languageModels.user` role"""
-    model_uri: str = ""
-    """Model uri to use."""
     folder_id: str = ""
     """Yandex Cloud folder ID"""
-    model_name: str = "text-search-query"
-    """Model name to use."""
+    doc_model_name: str = "text-search-doc"
+    """Doc model name to use."""
+    query_model_name: str = "text-search-query"
+    """Query model name to use."""
```

Comment:
this is a breaking change, we should find a way to make it backwards compatible

Reply:
updated, let me know what you think @egorkrash

Reply:
Oh yes, you are right @baskaryan 
This looks good, thanks for adding the test too!

Reply:
<img width="734" alt="image" src="https://github.com/langchain-ai/langchain/assets/20363212/61a9148f-eaa1-4157-8d98-0a8a53d884e7">

Maybe I'm looking in the wrong place, but this test seems to be skipped

Reply:
which job are you looking? that's probably the extended_test job, this runs in the regular make_test job (which doesn't require any optional dependencies) https://github.com/langchain-ai/langchain/actions/runs/8512241122/job/23313469252?pr=19720